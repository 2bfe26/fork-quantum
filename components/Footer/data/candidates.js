export default {
  id: 1,
  title: 'Candidatos',
  cols: 8,
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Buscar vagas',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Por palavra-chave',
      href: '/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Por área/cargo',
      href: 'https://www.catho.com.br/vagas/por-area/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Melhores e maiores empresas do Brasil',
      href: 'https://www3.catho.com.br/vag/empregos-maiores-empresas/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 2,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Produtos Gratuitos',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Aviso de vagas e buscas salvas',
      href: '/aviso-vagas/listagem/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Buscar vagas',
      href: 'https://www.catho.com.br/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Guia de Profissões e Salários',
      href: '/profissoes/',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Por dentro das empresas',
      href: '/por-dentro/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 3,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Apoio ao candidato',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Elaboração de Currículo',
      href: '/https://parceiros.catho.com.br/gop.php?origem=cat&a=6515&p=12&url1=1&url=servicoscatho%2Fcarreira%2Felaboracao-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Análise de Currículo',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6523&p=4&url1=1&url=servicoscatho%2Fcarreira%2Fanalise-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Simulação de Entrevista',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6527&p=16&url1=1&url=servicoscatho%2Fcarreira%2Fsimulacao-de-entrevista%2F',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Elaboração de Carta de Apresentação',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6519&p=13&url1=1&url=servicoscatho%2Fcarreira%2Fcarta-de-apresentacao%2F',
      target: '_self',
      img: null,
    }, {
      id: 6,
      title: false,
      text: 'Conheça Todas as Soluções',
      href: '/servicoscandidatos/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 4,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Carreira e sucesso',
      href: 'https://www.catho.com.br/carreira-sucesso/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 5,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Exclusivo para assinantes',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Cursos Rápidos',
      href: '/cursos-rapidos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Cadastro Catho',
      href: 'https://cadastro.catho.com.br/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Consultor Virtual',
      href: '/consvirtual/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 6,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Ajuda',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Central de Relacionamento',
      href: 'https://www.catho.com.br/sac/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: true,
      text: 'Promoção 7 dias grátis',
      href: null,
      target: null,
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Entenda a promoção',
      href: 'https://www.catho.com.br/help/usr/faq_promocao.phtml?home=1&width=650px&height=600px&iframe=true',
      target: '_self',
      img: null,
    }],
  }],
};
