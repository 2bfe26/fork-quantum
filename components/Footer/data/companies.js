export default {
  id: 1,
  title: 'Empresas',
  cols: 2,
  theme: {
    primaryColor: '#005C45',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: false,
    cols: 12,
    links: [{
      id: 1,
      title: false,
      text: 'Solução Atração dos Melhores',
      href: 'https://seguro.catho.com.br/inc/novaInclusao/FormPrincipal.php?tipoProd=Vag&c=13006',
      target: '_self',
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Solução Recrutamento Perfeito',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Produtos para Empresa',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }],
  }],
};
