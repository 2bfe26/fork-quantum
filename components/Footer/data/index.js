import Candidates from './candidates';
import Companies from './companies';
import Courses from './courses';
import Institutional from './institutional';
import Stamps from './stamps';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '15px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  candidates: Candidates,
  companies: Companies,
  courses: Courses,
  institutional: Institutional,
  stamps: Stamps,
};

export default Data;
