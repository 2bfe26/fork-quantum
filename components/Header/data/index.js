import Segments from './segments';
import Promotion from './promotion';
import Menu from './menu';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '10px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  segments: Segments,
  promotion: Promotion,
  menu: Menu,
};

export default Data;
