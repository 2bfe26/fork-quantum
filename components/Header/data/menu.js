const common = [
  {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Envie seu Currículo por 7 dias grátis',
    href: 'https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807',
    target: '_self',
    img: null,
  }, {
    id: 5,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 6,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 7,
    title: null,
    text: 'Portal do currículo',
    href: 'https://www.catho.com.br/portaldocurriculo/',
    target: '_self',
    img: null,
  }, {
    id: 8,
    title: null,
    text: 'Vagas de estágio',
    href: 'https://www.catho.com.br/estagio/',
    target: '_self',
    img: null,
  }, {
    id: 9,
    title: 'Exclusivo para assinantes',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 10,
    title: null,
    text: 'Cursos rápidos',
    href: '/cursos-rapidos/',
    target: '_self',
    img: null,
  }, {
    id: 11,
    title: null,
    text: 'Cadastro Catho',
    href: 'https://cadastro.catho.com.br/',
    target: '_self',
    img: null,
  }, {
    id: 12,
    title: null,
    text: 'Consultor virtual',
    href: '/consvirtual/',
    target: '_self',
    img: null,
  }, {
    id: 13,
    title: null,
    text: 'Vantagens de assinar a Catho',
    href: '/cadastrar-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 14,
    title: 'Apoio ao candidato',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 15,
    title: null,
    text: 'Elaboração de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 16,
    title: null,
    text: 'Análise de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 17,
    title: null,
    text: 'Simulação de entrevista',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/',
    target: '_self',
    img: null,
  }, {
    id: 18,
    title: null,
    text: 'Elaboração de carta de apresentação',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/',
    target: '_self',
    img: null,
  }, {
    id: 19,
    title: null,
    text: 'Cursos',
    href: '/curso/',
    target: '_self',
    img: null,
  }, {
    id: 20,
    title: null,
    text: 'Todas as soluções',
    href: '/servicoscandidatos/',
    target: '_self',
    img: null,
  },
];

const desktop = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    backgroundColor: '#FFFFFF',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '14px',
    marginLogo: '0px 20px 0px 0px',
    menuHover: {
      borderColor: '#005da4',
      borderSize: '2px',
    },
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Soluções para candidatos',
    href: null,
    target: null,
    img: null,
    collapsedLinkGroup: common,
  }],
};

const phone = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
    icon: 'search',
  }, {
    id: 2,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
    icon: 'enterprise',
  }, {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
    icon: 'alert',
  },
  ...common.slice(1)],
};

export default {
  desktop,
  phone,
};
