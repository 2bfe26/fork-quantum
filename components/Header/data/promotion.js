export default {
  theme: {
    primaryColor: '#da1d5d',
    secondaryColor: '#b6154e',
    textColor: '#fff',
  },
};
