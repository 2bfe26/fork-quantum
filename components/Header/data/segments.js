export default {
  id: 1,
  title: null,
  cols: 12,
  theme: {
    primaryColor: '#555',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'candidatos',
    href: '/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'empresas',
    href: 'https://www.catho.com.br/empresa/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: false,
    text: 'cursos',
    href: 'https://www.catho.com.br/educacao/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: null,
    href: 'https://www.catho.com.br/help/',
    target: '_self',
    img: {
      src: 'header-info.svg',
      width: '16px',
      height: '16px',
      alt: '',
    },
  }],
};
