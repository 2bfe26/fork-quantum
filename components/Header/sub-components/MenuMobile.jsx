import React from 'react';
import PropTypes from 'prop-types';
import styled, { ThemeProvider, css } from 'styled-components';
import { Container, Row, Col } from '../../Grid';

const MenuMobile = ({ data, menuOpened }) => (
  <ThemeProvider theme={data.theme}>
    <MenuMobileContainer fluid className={menuOpened}>
      <Row no-gutters>
        {
          data.menu.phone.links.map(link =>
            link.icon && (
              <Col phone={4} key={link.id}>
                <FeaturedListItem>
                  <FeaturedLink icon={link.icon} href={link.href} title={link.text}>
                    {link.text}
                  </FeaturedLink>
                </FeaturedListItem>
              </Col>
            ))
        }
        <LinkList>
          {
            data.menu.phone.links.map(link =>
              !link.icon && (
                <LinkListItem
                  key={link.id}
                  linkTitle={!!link.title}
                >
                  { link.text &&
                    <Link
                      href={link.href}
                      title={link.text}
                      target={link.target}
                    >
                      {link.text}
                    </Link>
                  }
                  {
                    link.title &&
                    link.title
                  }
                </LinkListItem>
            ))
          }
        </LinkList>
      </Row>
    </MenuMobileContainer>
  </ThemeProvider>
);

MenuMobile.propTypes = {
  data: PropTypes.instanceOf(Object),
  menuOpened: PropTypes.string,
};
MenuMobile.defaultProps = {
  data: null,
  menuOpened: null,
};

const MenuMobileContainer = styled(Container)`
  background-color: #FFF;
  box-sizing: border-box;
  font-family: ${props => props.theme.fontFamily};
  font-size: 11px;
  overflow-y: hidden;
  opacity: 1;
  padding: 0;
  position: fixed;
  transition: all 0.6s ease 0s;
  z-index:4;
  height: calc(100vh - 50px);
  top: -100vh;

  &.opened {
    animation: mobileMoveToViewHeight .7s forwards ease .25s;
    overflow-y: scroll;
  }

  &.closed {
    animation: mobileMoveToViewHeightReverse .7s ease-out;
  }

  @keyframes mobileMoveToViewHeightReverse {
    0% { transform: translateY(100vh); }
    100% { transform: translateY(0); }
  }

  @keyframes mobileMoveToViewHeight {
    100% { transform: translateY(100vh); }
  }
`;

const LinkList = styled.ul`
  list-style: none;
  margin: 0;
  padding: 0;
  width: 100%;
`;

const LinkListItem = styled.li`
  border-top: 1px solid #f5f5f5;
  color: #666;

  &:last-child {
    border-top-width: 3px;
  }

  ${props => props.linkTitle && css`
    background: #f5f5f5;
    font-size: 10px;
    height: 20px;
    line-height: 20px;
    text-indent: 20px;
    text-transform: uppercase;
  `}
`;

const Link = styled.a`
  border-left: 2px solid transparent;
  color: #666;
  display: block;
  padding: 18px 18px 17px;
  text-decoration: none;
  transition: all .15s ease-out;

  &:hover {
    border-left-color: #005da4;
  }
`;

const FeaturedListItem = styled.div`
  text-align: center;
  width: 100%;
`;

const FeaturedLink = styled.a`
  border-bottom: 2px solid transparent;
  color: #666;
  display: block;
  height: 125px;
  padding-top: 15px;
  text-decoration: none;
  transition: all .15s ease-out;

  &:hover {
    border-bottom-color: #005da4;
  }

  &:before {
    background: #005da4;
    border-radius: 50%;
    color: #fff;
    display: block;
    font-family: cathoIconFontB2c;
    font-smoothing: antialiased;
    font-size: 30px;
    font-weight: 400;
    height: 36px;
    width: 36px;
    left: 50%;
    line-height: 35px;
    margin-bottom: 15px;
    padding: 10px;
    position: relative;
    transform: translateX(-50%);
  }

  ${props => props.icon === 'search' && css`
    &:before {
      content: 'I';
    }
  `}

  ${props => props.icon === 'enterprise' && css`
    &:before {
      content: 'D';
    }
  `}

  ${props => props.icon === 'alert' && css`
    &:before {
      content: 'K';
    }
  `}
`;

export default MenuMobile;
