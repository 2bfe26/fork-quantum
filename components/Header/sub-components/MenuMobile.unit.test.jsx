import React from 'react';
import renderer from 'react-test-renderer';
import MenuMobile from './MenuMobile';
import data from '../data';

describe('MenuMobile component', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<MenuMobile data={data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
