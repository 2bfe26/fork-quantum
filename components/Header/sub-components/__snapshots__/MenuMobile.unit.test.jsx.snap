// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuMobile component Should match the snapshot 1`] = `
<div
  className="sc-ifAKCX kULZqa sc-bdVaJa OkJiS"
>
  <div
    className="sc-bwzfXH hnDOSg"
  >
    <div
      className="sc-htpNat hsrGhS"
    >
      <div
        className="sc-htoDjs RZnJw"
      >
        <a
          className="sc-dnqmqq cvhdim"
          href="/vagas/"
          icon="search"
          title="Buscar vagas"
        >
          Buscar vagas
        </a>
      </div>
    </div>
    <div
      className="sc-htpNat hsrGhS"
    >
      <div
        className="sc-htoDjs RZnJw"
      >
        <a
          className="sc-dnqmqq jYooTm"
          href="/por-dentro/"
          icon="enterprise"
          title="Por dentro das empresas"
        >
          Por dentro das empresas
        </a>
      </div>
    </div>
    <div
      className="sc-htpNat hsrGhS"
    >
      <div
        className="sc-htoDjs RZnJw"
      >
        <a
          className="sc-dnqmqq CjmeH"
          href="/aviso-vagas/listagem/"
          icon="alert"
          title="Aviso de vagas"
        >
          Aviso de vagas
        </a>
      </div>
    </div>
    <ul
      className="sc-EHOje ZUQmC"
    >
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807"
          target="_self"
          title="Envie seu Currículo por 7 dias grátis"
        >
          Envie seu Currículo por 7 dias grátis
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/profissoes/"
          target="_self"
          title="Guia de profissões"
        >
          Guia de profissões
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/por-dentro/"
          target="_self"
          title="Por dentro das empresas"
        >
          Por dentro das empresas
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://www.catho.com.br/portaldocurriculo/"
          target="_self"
          title="Portal do currículo"
        >
          Portal do currículo
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://www.catho.com.br/estagio/"
          target="_self"
          title="Vagas de estágio"
        >
          Vagas de estágio
        </a>
      </li>
      <li
        className="sc-bZQynM kBLwuG"
      >
        Exclusivo para assinantes
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/cursos-rapidos/"
          target="_self"
          title="Cursos rápidos"
        >
          Cursos rápidos
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://cadastro.catho.com.br/"
          target="_self"
          title="Cadastro Catho"
        >
          Cadastro Catho
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/consvirtual/"
          target="_self"
          title="Consultor virtual"
        >
          Consultor virtual
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/cadastrar-curriculo/"
          target="_self"
          title="Vantagens de assinar a Catho"
        >
          Vantagens de assinar a Catho
        </a>
      </li>
      <li
        className="sc-bZQynM kBLwuG"
      >
        Apoio ao candidato
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/"
          target="_self"
          title="Elaboração de currículo"
        >
          Elaboração de currículo
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/"
          target="_self"
          title="Análise de currículo"
        >
          Análise de currículo
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/"
          target="_self"
          title="Simulação de entrevista"
        >
          Simulação de entrevista
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/"
          target="_self"
          title="Elaboração de carta de apresentação"
        >
          Elaboração de carta de apresentação
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/curso/"
          target="_self"
          title="Cursos"
        >
          Cursos
        </a>
      </li>
      <li
        className="sc-bZQynM jpfThj"
      >
        <a
          className="sc-gzVnrw cDChpl"
          href="/servicoscandidatos/"
          target="_self"
          title="Todas as soluções"
        >
          Todas as soluções
        </a>
      </li>
    </ul>
  </div>
</div>
`;
